guides/common/modules/proc_viewing-image-mode-hosts-and-booted-container-images.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Viewing_image_mode_hosts_and_booted_container_images_{context}"]
+= Viewing image mode hosts and booted container images
+
+These actions streamline the management of image mode hosts, providing flexibility and control over host operations through {Project}.
+
+.Prerequisites
+* *Pull-Mode Remote Execution*: For pull-mode remote execution users, related packages must be included in the image mode host's container image.
```

Comment:
```suggestion
* *Pull-Mode Remote Execution*: If you use pull-mode remote execution, the `x`, `y`, and `z` packages must be included in the image mode host's container image.
```
Can we tell them what packages specifically they need?

Reply:
Adding a cross reference to https://docs.redhat.com/en/documentation/red_hat_satellite/6.16/html-single/managing_hosts/index#transport-modes-for-remote-execution_managing-hosts which covers this.